appveyor.yml

```diff
@@ -0,0 +1,40 @@
+version: 0.0.{build}
```

Comment:
I think the only thing needing a change is this - it should be `1.0.0.{build}` I think. The current release is `0.76.0` - let's go 1.0 with the release that has CI.